test/firrtl/lower-to-rtl-module.mlir

```diff
@@ -137,11 +137,11 @@
   }
 
   // expected-error @+1 {{cannot lower this port type to RTL}}
-  firrtl.module @CantLowerArgument(%arg: !firrtl.bundle<int_1: flip<uint<1>>, int_out: uint<2>>) {
+  firrtl.module @CantLowerArgument(%arg: !firrtl.bundle<int_1: flip<uint<1>>, int_out: uint<2>>) attributes {sym_visibility = "private"} {
   }   // CHECK-NEXT: }
 
   // expected-error @+1 {{unexpected operation 'func' in a firrtl.circuit}}
-  func @UnknownFunction() {
+  func @UnknownFunction() attributes {sym_visibility = "private"} {
```

Comment:
btw i think you can also use syntax like `func "private" @UnknownFunction ...` now. not sure if there is a preference from the owners here.

Reply:
Oh, I didn't know this. Cool! I'll defer to the owners since I am not really familiar with syntactic sugar in MLIR.

Reply:
Actually seems like the syntax is `func private @UnknownFunction ...` (no quotes around "private"). https://github.com/llvm/llvm-project/blob/b7382ed3fea08da27530a6d6d53f168fc704e4c4/mlir/test/IR/test-func-erase-result.mlir#L5

I would prefer the terser syntax myself. @lattner is the codeowner for all these test cases, and I don't want to speak for him, but I know he expressed interest on the [Discourse discussion](https://llvm.discourse.group/t/rfc-symbol-definition-declaration-x-visibility-checks/2140/12) in having barewords in these scenarios, so that may be the way to go.